compiler/parser/ceda/parse_to_json2.c

```diff
@@ -74,9 +76,11 @@ int main (int argc, char* argv []) {
         }
     }
 
+for (int i = 0; i < 1; i++) {
```

Comment:
?

Reply:
@thurston ?

Reply:
This is part of the C port. It does not affect the Python port.

I accidentally let through part of the instrumentation for benchmarking (here it parses once; I also time it parsing 100x, etc.).